templates/cluster-template-high-availabilty.yaml

```diff
@@ -0,0 +1,201 @@
+---
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: ${CLUSTER_NAME}
+spec:
+  clusterNetwork:
+    pods:
+      cidrBlocks: ["192.168.0.0/16"] # CIDR block used by Calico.
```

Comment:
I remember someone reported some issue on the pod conflict before 
maybe we can consider to make this configurable later on

Reply:
I think we have to decide which is mandatory configuration and what can be adjusted manual in the YAML. When you do the quickstart I think you should not be force to expose an env variable for everything. Right now we already need 10-20 env variables for OpenStack :/ That's why I though stuff like this should be changed manually after generating the cluster YAML. (same for node cidr). So not adding a var here only means you have to edit your YAML after generation and you are not *forced* to set the env var to generate your YAML

What do you think?

Reply:
um.. can this kind of env to be default value then anyone can feel free to change it later on? 
I mean, I am ok to ask folks to do the configuration by themselves but for those which might be affected , provide a default value and provide flexibility is good way 

we can merge your PR then talk later on :)

Reply:
I'm not sure how we can provide a default value. The only way would be to add it to the envrc

Reply:
ok, let's forget this, I will see what we can do after this PR merged

Reply:
Okay no problem :). It's just that we have a lot of properties and I don't want to force everyone to set all of them. I think we need some kind of basic set which have to be configured and everything else is editing the generated YAML (we need a good property documentation for this). Preferably the documentation generated in the CRDS which can be shown by kubectl explain is enough.

Reply:
@jichenjc Ya I faced this issue as my host ip is in the same range. But as @sbueringer says it might be better to just adjust it manually. I have no issues there.